src/components/Navbar/DesktopNavbar.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useEffect, useRef } from "react";
+import { styled } from "../../stitches.config";
+
+import Logo from "../../media/logo.png"
+import { TextLink } from "../Link";
+import { FiSearch } from "react-icons/fi"
+import { Link } from "gatsby";
+import { useState } from "react";
+//Must implement gatsby image plugin
+
+const Holder = styled("div", {
+  position: "relative",
+  margin: "0px",
+  width: "100%",
+  height: "100px",
+  backgroundColor: "#FFFEA1",
+  display: "flex",
+  alignItems: "center",
+})
+
+const PageLinks = styled("div", {
+  display: "flex",
+  alignItems: "center",
+  justifyContent: "space-around",
+  right: "100px",
+  position: "absolute"
+})
+
+const PageLink = styled(TextLink, {
+  margin: "15px"
+})
```

Comment:
Could you remove the underline from these links, as well as giving them the TextDark colour like in the design document?

I guess would then need to get them to swich colour to the lighter text colour on hover?